data/ecosystem/adopters.yaml

```
@@ -73,6 +73,11 @@
   components: [Collector, Go, Ruby]
   reference: ''
   contact: ''
+- name: Sicredi
+  url: https://www.sicredi.com.br/
+  components: [Collector, Go, Java, Python]
+  reference: ''
+  contact: ''
```

```suggestion
  reference: ''
  contact: ''
```

Do you have any blog post, conference talk (recording, slides, etc.) we can provide as reference?

for the contact, please put here either your github handle (preferred) or an email address we can reach out to if we have questions, e.g. if any link throws 404s or if we plan to apply significant changes.

---

Hey Severin, we have only internal material, but I'm working to publish a blog post about it. Can I update the reference soon?